tests/testthat/test-parser.R

```diff
@@ -0,0 +1,44 @@
+test_that("Using Perceval version 0.12.24", {
+  tools_path <- "../../tools.yml"
+  tool <- yaml::read_yaml(tools_path)
+  perceval_path <- tool[["perceval"]]
+  perceval_version <- system2(perceval_path, args="--version", stdout=TRUE, stderr=FALSE)
+  expect_equal(perceval_version, "perceval 0.12.24")
+})
+test_that("Correct perceval path", {
+  tools_path <- "../../tools.yml"
+  tool <- yaml::read_yaml(tools_path)
+  perceval_path <- tool[["perceval"]]
+  expect_equal(file.exists(perceval_path), TRUE)
+})
+test_that("Correct mbox path", {
+  conf_path <-"../../conf/thrift.yml"
+  conf <- yaml::read_yaml(conf_path)
+  mbox_path <- conf[["mailing_list"]][["mbox"]]
+  expect_equal(file.exists(mbox_path), TRUE)
+})
+test_that("Correct jira issues comments path", {
+  conf_path <-"../../conf/thrift.yml"
+  conf <- yaml::read_yaml(conf_path)
+  jira_issue_comments_path <- conf[["issue_tracker"]][["jira"]][["issue_comments"]]
+  expect_equal(file.exists(jira_issue_comments_path), TRUE)
+})
```

Comment:
Since this is another "does data exist" test, and we already test for the conf file, let's remove it.